codespell_lib/data/dictionary.txt

```diff
@@ -28988,6 +28988,8 @@ pre-defiend->pre-defined
 pre-defiened->pre-defined
 pre-empt->preempt
 pre-pended->prepended
+pre-poulate->pre-populate
+pre-poulated->pre-populated
```

Comment:
We usually add all verbal forms:
```
pre-poulates->pre-populates
pre-poulating->pre-populating
```

Reply:
Appreciate the suggestion! I've made this requested change.